proxy/Main.cc

```diff
@@ -381,8 +381,11 @@ class MemoryLimit : public Continuation
   {
     memset(&_usage, 0, sizeof(_usage));
     SET_HANDLER(&MemoryLimit::periodic);
+    RecRegisterStatInt(RECT_PROCESS, "proxy.process.traffic_server.memory.rss", (RecInt)0, RECP_NON_PERSISTENT);
```

Comment:
No C-style casts!

Reply:
oh no! C-style casts? @zwoop you do understand what that means. Don't you?